android/expoview/src/main/java/versioned/host/exp/exponent/ExperiencePackagePicker.kt

```diff
@@ -75,7 +75,6 @@ object ExperiencePackagePicker : ModulesProvider {
     ConstantsPackage(),
     ContactsPackage(),
     FaceDetectorPackage(),
-    FileSystemPackage(),
```

Comment:
we still need this FileSystemPackage because other internal modules like FileSystemPermissionModule are not migrated.

Reply:
Reverted